app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Command should be in the format:"
+                         " 'mv source_file destination_path'")
+
+    source, destination = parts[1], parts[2]
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f'The source file "{source}" does not exist.')
+
+    if destination.endswith("/") or (os.path.isdir(destination)
```

Comment:
Do not use any specific directory delimiters, your code should be platform-agnostic